src/components/ui/sidebar/nav-user.tsx

```diff
@@ -27,7 +27,11 @@ import useAuthUser, { useAuthContext } from "@/hooks/useAuthUser";
 import { usePatientSignOut } from "@/hooks/usePatientSignOut";
 import { usePatientContext } from "@/hooks/usePatientUser";
 
-export function FacilityNavUser() {
+export function FacilityNavUser({
+  selectedFacilityId,
+}: {
+  selectedFacilityId?: string;
```

Comment:
is it really optional?

```suggestion
  selectedFacilityId: string | undefined;
```

Reply:
yea 🤔  optional (all we are using it is to check if it exists and render profile url accordingly anyways)